src/routes/connect/connectHelpers.ts

```diff
@@ -16,3 +16,19 @@ export async function decryptPrivatePair(): Promise<string> {
         throw new StorageKeyNotFoundError();
     }
 }
+
+export function toFindDuplicates(arry: string[]): string[] {
+    return arry.filter((item, index) => arry.indexOf(item) !== index);
+}
+
+export function getParamsFromUrl() {
```

Comment:
this is not getting all the parameters from the url, it's specifically getting wallets, so the name should reflect that